src/views/StudyFilesListView.js

```diff
@@ -62,27 +66,74 @@ const StudyFilesListView = ({
   const files = !loading ? studyByKfId.files.edges : [];
   return (
     <Grid as={Segment} basic container columns={1}>
-      <Grid.Column width={16}>
-        <Header as="h2">Upload Study Documents for DRC Approval</Header>
-        {loading ? (
-          <StudyListSkeleton />
-        ) : (
-          <FileList fileList={files} studyId={kfId} />
-        )}
-        <Divider />
-        <Grid.Row>
-          <UploadContainer
-            handleUpload={file =>
-              history.push('documents/new-document', {file})
-            }
-          />
-        </Grid.Row>
-      </Grid.Column>
-    </Grid>
+      <Grid.Row>
+        <Grid.Column width={10}>
+          <h2 inline>Study Documents</h2>
+        </Grid.Column>
+        {
+          files.length ? <Grid.Column width={6}>
```

Comment:
Instead of doing a ternary with a null `false`, you could do a `{files.length && <Grid>{stuff}</Grid>}`